components/backup/src/endpoint.rs

```diff
@@ -575,9 +573,10 @@ impl<E: Engine, R: RegionInfoProvider> Endpoint<E, R> {
         &self,
         prs: Arc<Mutex<Progress<R>>>,
         request: Request,
-        tx: mpsc::Sender<(BackupRange, Result<BackupRes>)>,
+        tx: UnboundedSender<BackupResponse>,
```

Comment:
can we use a bounded channel here to prevent that potential OOM? @overvenus

Reply:
I don't think so, sending a response is very quick, it may not block any thread in practical.